client/components/cloud/project/billing/outgoing-trafic/outgoing-trafic.component.controller.js

```diff
@@ -0,0 +1,58 @@
+class OutgoingTraficComponentCtrl {
+  constructor($translate, ControllerHelper, CLOUD_GEOLOCALISATION,
```

Comment:
Please order your services this way : 

- AngularJS Provider ($http etc..)
- Services/Provider ...
- Constants

As an example it could be like this : 

```suggestion
  constructor($translate, ControllerHelper, OvhApiMe, RegionService, ServiceHelper, CLOUD_GEOLOCALISATION)
```